Content.Shared/_RMC14/BulletBox/BulletBoxSystem.cs

```diff
@@ -35,18 +40,40 @@ private void OnExamined(Entity<BulletBoxComponent> ent, ref ExaminedEvent args)
         }
     }
 
+    private void OnGetAlternativeVerbs(Entity<BulletBoxComponent> ent, ref GetVerbsEvent<AlternativeVerb> args)
+    {
+        var user = args.User;
+        var verb = new AlternativeVerb()
+        {
+            Act = () => { ent.Comp.TransferToBox = !ent.Comp.TransferToBox;
+                if (_net.IsServer)
+                {
+                    _popup.PopupEntity(Loc.GetString("rmc-bullet-box-refilling-" + ent.Comp.TransferToBox.ToString()), ent, user);
```

Comment:
The popup to display should be two LocId fields on the component, not concatenating it like this